src/App.jsx

```diff
@@ -14,33 +17,164 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_ALPHABET = 'Sort alphabetically';
+const SORT_LENGTH = 'Sort by length';
+const REVERSE = 'Reverse';
+const RESET = 'Reset';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function styleClass(btn) {
+  switch (btn) {
+    case SORT_ALPHABET:
+      return 'is-info';
+    case SORT_LENGTH:
+      return 'is-success';
+    case REVERSE:
+      return 'is-warning';
+    case RESET:
+      return 'is-danger';
+    default:
+      return '';
+  }
+}
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+export const App = () => {
+  const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
+  const [style, setStyle] = useState({
+    SORT_ALPHABET: false,
+    SORT_LENGTH: false,
+    REVERSE: false,
+  });
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  function changeList(btn) {
```

Comment:
```suggestion
  function sortGoods(goods, { sortType, isReversed }) {
```
- Move function that sorts and reversing outside the component
- And the name is confusing
- It should be a pure function that just prepares goods in accordance with parameters and returns prepared